### DISCUSSION TITLE
Ability to merge multiple GPU fans into a single control

### DISCUSSION THREAD

**Discussion by VeilSide47** [Rem0o/FanControl.Releases#2215] [Ideas]
As the title says. My GPU has 3 fans. FanControl detects 2 side fans and the middle one as separate. I always run them on the same curve/RPM. Would be nice to have an option to be able to merge them all into a single control.

**Comment by Rem0o** [comment]
Use a sync curve to sync 1 control with the other, then hide the control/curve from the UI.

**Comment by VeilSide47** [comment]
This worked wonderfully. Many thanks.
